.devops/deploy-pipelines.yml

```
@@ -575,4 +886,72 @@ stages:
+    dependsOn: Rollback_New_Deployment_Helm_Release
+    pool: server
+    jobs:
+      - job: Start_New_Stop_Staging_Receivers_Manual_Approve_Rollback
+        displayName: Approve start PROD deployment receivers and stop STAGING for rollback processing (swap back deployments)
+        timeoutInMinutes: 30
+        steps:
+          - task: ManualValidation@0
+            timeoutInMinutes: 30
+            inputs:
+              notifyUsers: $(APPROVE_TOUCHPOINT_MAIL)
+              instructions: "Approve enable PROD deployment receivers and stop STAGING ones for deploy rollback (swap back)"
+              onTimeout: 'reject'
+  - stage: Start_New_Stop_Staging_Receivers_Rollback
+    displayName: Start PROD receivers and stop STAGING one approval for Rollback
+    dependsOn: Start_New_Stop_Staging_Receivers_Manual_Approve_Rollback
+    pool:
+      name: pagopa-prod-linux
+    jobs:
+      - job: Start_New_Stop_Staging_Receivers_Rollback
+        displayName: Start PROD receivers and stop STAGING for rollback processing (swap back deployments)
+        steps:
+          - template: azure-templates/start-receivers.yml
+            parameters:
+              ENV: PROD
+              TARGET_DEPLOYMENT_VERSION: PROD
+          - template: azure-templates/stop-receivers.yml
+            parameters:
+              ENV: PROD
+              TARGET_DEPLOYMENT_VERSION: STAGING
+  # STEP 4: scale down staging release (by adding back annotation to scaledObject)
+  - stage: Scale_Down_Staging_Deployment_Approval_Rollback
+    dependsOn: Start_New_Stop_Staging_Receivers_Rollback
+    displayName: Approval for scale down STAGING deployment PODS to 0
+    pool: server
+    jobs:
+      - job: Scale_Down_Staging_Deployment_Approval
+        timeoutInMinutes: 30
+        steps:
+          - task: ManualValidation@0
+            timeoutInMinutes: 30
+            inputs:
+              notifyUsers: $(APPROVE_TOUCHPOINT_MAIL)
+              instructions: "Approve scale STAGING deployment down to 0 pods"
+              onTimeout: 'reject'
+  - stage: Scale_Down_Staging_Deployment_Rollback
+    dependsOn: Scale_Down_Staging_Deployment_Approval_Rollback
+    displayName: Scale down STAGING deployment to 0 pods
+    pool:
+      name: pagopa-prod-linux
+    jobs:
+      - job: Scale_Down_Staging_Deployment_Rollback
+        displayName: Annotate staging release scaledObject with autoscaling.keda.sh/paused-replicas="0"
+        steps:
+          - task: Kubernetes@1
+            displayName: Scale down staging receivers PODS (annotate ScaledObject with autoscaling.keda.sh/paused-replicas="0")
+            continueOnError: true
+            inputs:
+              kubernetesServiceEndpoint: $(PROD_KUBERNETES_SERVICE_CONN)
+              kubernetesCluster: $(kubernetesCluster)
+              namespace: ecommerce
+              command: scale
+              arguments: --replicas=0 deployment/pagopaecommercetxschedulerservice-microservice-chart
```

watch out! the deployment to be scaled down is the staging one
```suggestion
              arguments: --replicas=0 deployment/$(K8S_IMAGE_REPOSITORY_NAME)-staging-microservice-chart
```

---

Resolved by https://github.com/pagopa/pagopa-ecommerce-transactions-scheduler-service/pull/162/commits/d346a7b1a358099583b221804054f19ebb1b16d3